Geometry/data/testbeam/gem11.xml

```diff
@@ -844,13 +844,13 @@
   <PosPart copyNumber="1">
     <rParent name="gem11:GSAX11L"/>
     <rChild name="gem11:GHA101"/>
-    <Translation x="0*fm" y="0*mm" z="0*cm" />
+    <Translation x="0*fm" y="-351*mm" z="0*cm" />
   </PosPart>
   <PosPart copyNumber="2">
     <rParent name="gem11:GSAX11L"/>
     <rChild name="gem11:GHA102"/>
     <rRotation name="x180y90" />
-    <Translation x="0*fm" y="1*mm" z="0*cm" />
+    <Translation x="0*fm" y="-350*mm" z="0*cm" />
```

Comment:
does this still work with geant?

Reply:
Yep, saw the rechits from the simulation.

Reply:
are u sure the hits are from this chamber?

Reply:
can u answer this

Reply:
doesnt this makes the gem 10x10 out of the gembox?

Reply:
<img width="875" alt="image" src="https://user-images.githubusercontent.com/25997327/142061325-4ee95519-340c-4d39-93a1-ca0f9481951d.png">
This image shows the rec hit position with the simulation.
It works.